cachi2/core/package_managers/bundler/parser.py

```diff
@@ -38,19 +40,32 @@ class _GemMetadata(pydantic.BaseModel):
     name: str
     version: str
 
+    def download_to(self, fs_location: Path) -> None:
+        return None
+
 
 class GemDependency(_GemMetadata):
     """
     Represents a gem dependency.
 
     Attributes:
-        source:     The source URL of the gem.
+        source:     The source URL of the gem as stated in 'source' field from Gemfile.lock.
```

Comment:
Actually, it is 'remote'. The problem is that the bundler uses 'remote' also for git and path dependencies.

Reply:
Huh, it is 'source' in Gemfile, but 'remote' in Gemfile.lock. Will update the comment.